.devcontainer/Dockerfile

```diff
@@ -1,19 +1,23 @@
-FROM python:3.10
+FROM python:3.14
 
 WORKDIR /usr/src/mobimart
 
-RUN apt-get update
+# Install Mise for the same tool management as Netlify
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+ENV MISE_DATA_DIR="/mise"
+ENV MISE_CONFIG_DIR="/mise"
+ENV MISE_CACHE_DIR="/mise/cache"
+ENV MISE_INSTALL_PATH="/usr/local/bin/mise"
+ENV PATH="/mise/shims:$PATH"
+COPY .mise.toml /etc/mise/config.toml
+RUN curl https://mise.run | sh
```

Comment:
we tried a few different techniques to lay down Node.js and Python side by side. i'm fond of mise because it allows for managing the version of terraform, python and other tools that we use centrally.

one other benefit is that its what netlify uses under the hood too.

that said, I wouldn't be fussed to use `nvm` or something else entirely instead.